### PR TITLE
feat: post validation

### PR DESCRIPTION
**Pull request by e55o**

### CODE REVIEW

File: lib/controllers/service/servicePostCtrl.js [hunk 1]
```diff
@@ -1,31 +1,22 @@
 const axios = require('axios');
 const errorUtil = require('../../errors/utils');
 
+const postServiceData = requestBody =>
+  axios.post('http://localhost:9999/service100', requestBody).then(() => {
+    if (requestBody) {
+      return requestBody;
+    }
+    throw errorUtil.makeError('No data from server', 500);
```
Comment (george-haddad) [inline lib/controllers/service/servicePostCtrl.js]: Can you research what would be the proper status code to return if you got a response with no request body?

File: lib/routes/service.js [hunk 2]
```diff
@@ -17,11 +17,11 @@ function service(router) {
   router.get('/', _.partial(serviceGetCtrl.getServiceRequest));
   router.get('/:id', _.partial(serviceGetCtrl.getServiceRequest));
   router.post('/', validate(serviceValidation.postService), _.partial(servicePostCtrl.postServiceRequest));
-  router.put(
-    '/:id',
-    validate(bodyValidation.putService),
-    _.partial(servicePutCtrl.putServiceRequest),
-  );
+  // router.put(
```
Comment (george-haddad) [inline lib/routes/service.js]: No commented code, please remove.

Comment (e55o) [reply lib/routes/service.js]: this is the part of Ralph that was causing error in master

File: lib/controllers/service/servicePostCtrl.js (hunk 1 above)
Comment (e55o) [reply lib/controllers/service/servicePostCtrl.js]: i think it is either 500 or 501, this is the [reference](https://github.com/george-haddad/api-proxy/pull/25#pullrequestreview-100373491) i used

File: lib/routes/service.js (hunk 2 above)
Comment (george-haddad) [reply lib/routes/service.js]: remove it anyways, I'll deal with it as well on the other pr

File: lib/controllers/service/servicePostCtrl.js (hunk 1 above)
Comment (george-haddad) [reply lib/controllers/service/servicePostCtrl.js]: 500 it is, lets also change the response message to properly match the error that is occurring too

Comment (e55o) [reply lib/controllers/service/servicePostCtrl.js]: should it be something like `Empty Request Body` ?

Comment (george-haddad) [reply lib/controllers/service/servicePostCtrl.js]: or `Unexpected empty body in request`

**Pull request merged**